docs/Reference/Pantheon-CLI-Syntax.md

```diff
@@ -1009,6 +1030,34 @@ receipt. Default is `false`.
     Enabling revert reason may use a significant amount of memory. We do not recommend enabling revert
     reason when connected to public Ethereum networks. 
 
+### remote-connections-limit-enabled
+
+```bash tab="Syntax"
+--remote-connections-limit-enabled[=<true|false>]
+```
+
+```bash tab="Command Line"
+--remote-connections-limit-enabled=false
+```
+
+```bash tab="Environment Variable"
+PANTHEON_REMOTE_CONNECTIONS_LIMIT_ENABLED=false
+```
+
+```bash tab="Configuration File"
+remote-connections-limit-enabled=false
+```
+
+Specify to limit the percentage of wire connections initiated by peers. Default is true. 
```

Comment:
If we want to be consistent with the CLI help, that text is currently:
```suggestion
Whether to limit the number of P2P connections initiated remotely. Default is true. 
```

Reply:
The CLI help isn't consistent with CLI help